ui/intermittent-failures/GraphAlternateView.jsx

```diff
@@ -39,20 +40,20 @@ const GraphAlternateView = ({ className, graphData, colNum, title }) => {
     alternateGraph.data = graphData[0].data;
   } else {
     alternateGraph.column = columnsThree;
-    // iterate between different types of data
-    graphData.forEach(graphItem => {
-      // iterate each data point
-      graphItem.data.forEach((item, index) => {
-        const { date, failureCount, pushCount } = item;
-        // populate array with data points, with combined properties
-        alternateGraph.data[index] = {
+    // create new array with objects with combined properties
+    alternateGraph.data = zipWith(
+      graphData[0].data,
+      graphData[1].data,
+      (a, b) => {
+        const { date, failureCount } = a;
```

Comment:
Do we know for certain these two arrays are in order by date? It might be a good precaution to sort each by date first before trying to match them in this way.

Reply:
I sorted the data in the `helpers.js` file. I thought it would be better to deliver the data ready to the component to render.

Reply:
Actually, I think this isn't necessary. In the `calculateMetrics` function, the objects containing the date are all inserted at the same time and nothing is changing that order so it should be fine without this.